File: pythonp/loops/book.py
class Book:
    caseHandling = 0

    def loop(self):
        for x in range(10):
            print("Loop::", x)
        print("Fixed::Loop::Completed")

    def whileLoop(self):
        i = 0
        while i < 10:
            i += 1
            print("*********************")
        print("While::Loop::Completed")


b1 = Book()
b1.loop()
b1.whileLoop()

File: javascriptp/loops/book.js
var Book = function () {
  this.condition = "a";
  this.loop = function () {
    for (var i = 0; i < 10; i++) {
      console.log("Loop::", i);
    }
  };
  this.while = function () {
    var i = 0;
    while (i < 10) {
      i++;
      console.log("*******************");
    }
    console.log("While loop completed");
  };
  this.doWhile = function () {
    var i = 0;
    do {
      i++;
      console.log("*******************");
    } while (i < 10 && i > 0);
    console.log("Do while loop completed");
  };
};

const newBook = new Book();
newBook.loop();
newBook.while();
newBook.doWhile();
